kubernetes/zenko/charts/opal/values.yaml

```diff
@@ -73,7 +73,7 @@ blobserver:
         schedule: '*/5 * * * *'
 
   image:
-    repository: registry.scality.com/sf-eng/blobserver
+    repository: registry.scality.com/scality/blobserver
```

Comment:
should this be `zenko`?

Reply:
`zenko` can only host public images, so we are using `scality` namespace to host private ones.